.prettierignore

```diff
@@ -0,0 +1 @@
+tests/fixtures/*.md
```

Comment:
The fixtures are a markdown file which prettier auto corrects, we don't want that.